packages/amplify-provider-awscloudformation/src/__tests__/iterative-deployment/stack-event-monitor.test.ts

```diff
@@ -0,0 +1,41 @@
+import { StackEventMonitor, IStackProgressPrinter } from '../../iterative-deployment/stack-event-monitor';
+import { CloudFormation } from 'aws-sdk';
+
+const stackProgressPrinterStub = ({
+  print: jest.fn(),
+  start: jest.fn(),
+  stop: jest.fn(),
+} as unknown) as IStackProgressPrinter;
+
+const cfn = ({
+  describeStackEvents: () => ({
+    promise: () => ({
+      NextToken: undefined,
+    }),
```

Comment:
Shouldn't this call `Promise.resolve()` so that the mocked behavior is closer to the real behavior.

Reply:
Yep, that makes more sense. Made the changes 👍🏻